### PR TITLE
Project Movies

### PR DESCRIPTION
**Pull request by JHarde**
Link to my deployed project:

https://app.netlify.com/sites/nostalgic-einstein-d16863/deploys

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

**Review by Bluepoisson — APPROVED** [review]
Great job Joel! Clean and easy to follow code, very cool with the dropdown menu that allows to chose from different lists. Easy navigation and excellent use of the API with clickable production companies that takes user further 💯 Would have been nice with a back button there as well :) Keep ut the awesome work ⭐

File: code/src/components/LoadingSpinner.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import { makeStyles } from "@material-ui/core/styles";
+import CircularProgress from "@material-ui/core/CircularProgress";
+
+const useStyles = makeStyles((theme) => ({
```
Comment (EmOlsz) [inline code/src/components/LoadingSpinner.js:6]: south-park-nice

File: code/src/components/MovieThumb.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// import React from "react";
```
Comment (EmOlsz) [inline code/src/components/MovieThumb.js:1]: Try to remember to remove commented code before final push next time!

File: code/src/pages/CompanyInfo.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      .then((json) => {
+        if (json.status_code === 34) {
+          setError(true);
+        } else {
+          setCompany(json);
+        }
+        setLoading(false);
+      });
+  }, [companyId]);
+  console.log(company);
```
Comment (EmOlsz) [inline code/src/pages/CompanyInfo.js:28]: Same goes to console.logs

**Pull request closed without merging**